src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout_actions.tsx

```diff
@@ -34,11 +35,36 @@ export interface DiscoverGridFlyoutActionsProps {
 }
 
 export function DiscoverGridFlyoutActions({ flyoutActions }: DiscoverGridFlyoutActionsProps) {
+  const { euiTheme } = useEuiTheme();
+  const [ref, setRef] = useState<HTMLDivElement | HTMLSpanElement | null>(null);
+  const dimensions = useResizeObserver(ref);
+  const isMobileScreen = useIsWithinBreakpoints(['xs', 's']);
+  const isLargeScreen = dimensions?.width ? dimensions.width > euiTheme.base * 30 : false;
```

Comment:
```suggestion
  const isLargeFlyout = dimensions?.width ? dimensions.width > euiTheme.base * 30 : false;
```
Nit: I agree with @kertal  that `isLargeFlyout` might be more appropriate for this.